pygfx/geometries/_base.py

```diff
@@ -82,6 +85,36 @@ def __init__(self, **kwargs):
             # Store
             setattr(self, name, resource)
 
+    @property
+    def morph_attributes(self):
```

Comment:
I am opposed to adding this property to the generic `Geometry` object, because it's rather specific, even for meshes, and it means nothing for e.g. a `Line` object.

I think there's plenty of options though:
1. A custom `MeshGeometry` or `MeshMorphGeometry`.
2. Just put the `morph_attributes` in the geometry in `load_gltf`. We do this for other data too, so why not for this.
3. Put `morph_positions`, `morph_normals`, and `morph_colors` separately on the geometry.

I think my preference would be 3, since it's transparent and flat.

Reply:
> and it means nothing for e.g. a `Line` object

Morph targets are meaningful for `Line` and `Point` objects as well. Essentially, they record the 'vertex attribute transformation methods,' enabling vertex attribute deformation animations even for `Line` and `Point` objects.

I realize that perhaps I should put the shader logic related to morph targets into `BaseShader` rather than `MeshShader`. 😅 

Moreover, theoretically, any vertex attribute can change, not just position, color, and normal; for instance, texcoord can also vary. 
However, currently, we only support morph of position, color, and normal attributes, as these are the most common vertex deformations. so I think 3 is OK.